app/types.ts

```diff
@@ -0,0 +1,10 @@
+interface Vault {
```

Comment:
```suggestion
interface DApp {
```

Reply:
Done. I have done the other changes required for this.